### PR TITLE
[11.0][FIX] Add Bank Account info + Hide Unnecessary fields

### PR DESCRIPTION
**Pull request by jarroyomorales**
Pues eso

### CODE REVIEW

**Comment by jarroyomorales** [comment]
Hablar sobre bank account antes de mergear

File: cb_hr_views/models/hr_attendance.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import fields, models
+
+
+class HrAttendance(models.Model):
+    _inherit = "hr.attendance"
+
+    center_id = fields.Many2one(
+        string='Center',
```
Comment (etobella) [inline cb_hr_views/models/hr_attendance.py]: readonly=True, store=True

File: cb_hr_views/views/hr_attendance_module_views.xml [hunk 2]
```diff
@@ -13,4 +13,16 @@
...
         <field name="groups_id" eval="[(6, 0, [ref('hr_attendance.group_hr_attendance')])]"/>
     </record>
+
+    <record id="hr_attendance_view_filter" model="ir.ui.view">
+        <field name="name">hr.attendance.search</field>
+        <field name="model">hr.attendance</field>
+        <field name="inherit_id" ref="hr_attendance.hr_attendance_view_filter"/>
+        <field name="arch" type="xml">
+            <field name="department_id" position="after">
+                <field name="center_id"/>
```
Comment (etobella) [inline cb_hr_views/views/hr_attendance_module_views.xml:23]: Probably will not work if not stored...

**Review by etobella — CHANGES_REQUESTED** [review]

**Comment by jarroyomorales** [comment]
@etobella This is RTM

**Review by etobella — APPROVED** [review]

**Pull request merged**